test/unit/object/test_item.py

```diff
@@ -25,6 +25,21 @@ def test_update_info(test_item_and_response, mock_box_session, etag, if_match_he
     assert update_response.object_id == test_item.object_id
 
 
+def test_update_info_with_default_request_kwargs(test_item_and_response, mock_box_session, mock_box_session_2):
+    # pylint:disable=redefined-outer-name, protected-access
+    test_item, mock_item_response = test_item_and_response
+    expected_url = test_item.get_url()
+    mock_box_session.with_default_network_request_kwargs.return_value = mock_box_session_2
+    mock_box_session_2.put.return_value = mock_item_response
+    data = {'foo': 'bar', 'baz': {'foo': 'bar'}, 'num': 4}
+    extra_network_parameters = {'timeout': 1}
+    update_response = test_item.update_info(data, extra_network_parameters=extra_network_parameters)
+    mock_box_session.with_default_network_request_kwargs.assert_called_once_with({'timeout': 1})
```

Comment:
Can just pass `extra_network_parameters` here, instead of re-constructing the `dict`.